### PR TITLE
New package: elixir-1.1.0beta

### PR DESCRIPTION
**Pull request by manveru**

### CODE REVIEW

File: srcpkgs/elixir/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'elixir'
+homepage="http://elixir-lang.org"
+license="Apache-2.0"
+maintainer="Michael 'manveru' Fellinger <mf@seitenschmied.at>"
+pkgname=elixir
```
Comment (xtraeme) [inline srcpkgs/elixir/template]: `pkgname`, `version` and `revision` must be the first three variables appearing in a template.

File: srcpkgs/elixir/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+version=1.1.0beta
+revision=1
+short_desc="Dynamic, functional language for scalable and maintainable applications"
+
+hostmakedepends="erlang git"
+depends="erlang>=17.0"
+distfiles="https://github.com/elixir-lang/elixir/archive/v1.1.0-beta.tar.gz"
+checksum="a506907137dc2432bf40f3c6a86ca807af5bf9d7f3a3efd05fed14e5267beb79"
+wrksrc="elixir-1.1.0-beta"
+only_for_archs="x86_x64 i686 armv6hf armv7hf"
```
Comment (xtraeme) [inline srcpkgs/elixir/template]: There's no reason to force `only_for_archs`, remove this.

File: srcpkgs/elixir/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+do_build() {
```
Comment (xtraeme) [inline srcpkgs/elixir/template]: Remove this `LC_ALL` export, it's unnecessary.

File: srcpkgs/elixir/template (hunk 2 above)
Comment (manveru) [inline srcpkgs/elixir/template]: Then the build fails on the other architectures, I already tried without when I first made the PR.

File: srcpkgs/elixir/template (hunk 3 above)
Comment (manveru) [inline srcpkgs/elixir/template]: Then the build will complain, since LC_ALL is C in the chroot.

File: srcpkgs/elixir/template (hunk 1 above)
Comment (manveru) [inline srcpkgs/elixir/template]: How come none of the docs or even xlint mention that requirement?

File: srcpkgs/elixir/template (hunk 2 above)
Comment (xtraeme) [inline srcpkgs/elixir/template]: If cross compilation fails there's no reason to force a specific list of architectures, use `nocross=yes` instead.

File: srcpkgs/elixir/template (hunk 3 above)
Comment (xtraeme) [inline srcpkgs/elixir/template]: That was fixed long ago, it's unnecessary.

**Comment by xtraeme** [comment]
Duplicate of #2839

**Pull request closed without merging**